assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/PriceModifierActions.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import AddPriceModifierButton from './AddPriceModifierButton';
+import DeletePriceModifierButton from './DeletePriceModifierButton';
+
+import { Price } from '../../data/types';
+import { TicketPriceCalculatorMutators } from './types';
+
+interface PriceModifierActionsProps {
+	index: number;
+	name: string;
+	price: Price;
+	modifierOptions: Price[];
+	fields: TicketPriceCalculatorMutators;
+}
+
+const PriceModifierActions = ({
+	index,
+	name,
+	price,
+	modifierOptions,
+	fields: { push, remove, reset, sort },
+}: PriceModifierActionsProps): React.ReactNode => {
+	const actions = [];
+	const key = `${price.id}-${index}`;
+	if (price.id === 'NEW_PRICE') {
+		actions.push(
+			<AddPriceModifierButton
+				key={key}
+				modifiers={modifierOptions}
+				name={name}
+				price={price}
+				push={push}
+				reset={reset}
+				sort={sort}
+			/>
+		);
+	} else if (!price.isBasePrice) {
+		actions.push(<DeletePriceModifierButton key={key} index={index} remove={remove} />);
+	}
```

Comment:
These conditionals might need to be simplified or maybe buttons might be added in other component.

Reply:
how would you simplify them any more than they are? (not being snarky, I'm sincerely interested)

> or maybe buttons might be added in other component.

that's exactly what this component does... originally this logic was inside the `TicketPriceModifierRow` component, but now it just appears as another component within the JSX like:

```jsx
<td style={{ ...styles.colWidth7h, ...styles.actions }}>
	<PriceModifierActions
		fields={fields}
		index={index}
		modifierOptions={modifierOptions}
		name={name}
		price={price}
	/>
</td>
```

and the only thing that `PriceModifierActions` does is add these buttons